app/models/tag.rb

```diff
@@ -47,6 +44,10 @@ class Tag < ActsAsTaggableOn::Tag
   after_commit :sync_related_elasticsearch_docs, on: [:update]
   after_commit :remove_from_elasticsearch, on: [:destroy]
 
+  pg_search_scope :search_by_name,
+                  against: :name,
+                  using: { tsearch: { prefix: true } }
+
```

Comment:
only moved it down closer to the "scopes" section, after all, it's an Active Record scope as well